### PR TITLE
feature: preferences quickstart

### PR DESCRIPTION
**Pull request by JEverhart383**
### Description
Goal of this PR is to help the customer move more quickly in implementing a preferences center, and will touch a few different parts of the docs and different pages to support that task:

- [x] Reorganize 'overview' page to include conceptual steps of creating a preference center, e.g. creating default prefs., getting prefs., rendering UI, setting prefs.
- [x] Revise existing in-app UI doc into a quickstart guide with steps, etc.
- [ ] Record and link video to support deeper understanding of Knock's pref model

### CODE REVIEW

File: content/in-app-ui/react/preferences.mdx [hunk 1]
```diff
@@ -57,127 +70,516 @@ In this example we'll use a `PreferenceSet` that contains multiple workflows, ea
...
+  height={714}
+  className="rounded-md mx-auto border border-gray-200"
+/>
+
+</Step>
+</Steps>
+
+## Completed preference center
+
+Knock's preference model is very flexibile, but you should find that the component below will satisfy most of your preference center needs and can easily be used as a starting point for your own preference center.
```
Comment (samseely) [inline content/in-app-ui/react/preferences.mdx:426]:
```suggestion
Knock's preference model is very flexible, but you should find that the component below will satisfy most of your preference center needs and can easily be used as a starting point for your own preference center.
```

**Review by samseely — APPROVED** [review]
Great stuff. Found one small typo but otherwise g2g 🚀

File: content/preferences/overview.mdx [hunk 2]
```diff
@@ -72,13 +74,30 @@ The `PreferenceSet` above models this preference grid in your application:
...
 />
 
+<Callout
+  mt={4}
+  emoji="👩‍💻"
+  text={
+    <>
+      <span className="font-bold">See an example.</span> Check out{" "}
+      <a
+        href="https://knock-in-app-notifications-react.vercel.app/preferences"
```
Comment (rachael-t) [inline content/preferences/overview.mdx:84]: Maybe we should update this to the custom domain versus the Vercel one: https://in-app-demo.knock.app/

**Review by rachael-t — APPROVED** [review]
This is amazing and will be so helpful for customers!

**Pull request merged**